### PR TITLE
test greenboot ci

### PR DESCRIPTION
**Pull request by henrywang**

### CODE REVIEW

**Comment by henrywang** [comment]
/greenboot-rs-test-rawhide say-paul/images:remove-old-greenboot-services

**Comment by henrywang** [comment]
/greenboot-rs-test-rawhide say-paul/images:remove-old-greenboot-services

**Comment by henrywang** [comment]
/greenboot-rs-test-rawhide say-paul/images:remove-old-greenboot-services

**Comment by henrywang** [comment]
/greenboot-rs-test-rawhide say-paul/images:remove-old-greenboot-services

**Comment by yih-redhat** [comment]
/greenboot-rs-test-rawhide say-paul/images:remove-old-greenboot-services

File: greenboot-test/greenboot-rs.sh [hunk 1]
```diff
@@ -32,7 +32,7 @@ SSH_KEY=key/ostree_key
 
 # Set os-variant and boot location used by virt-install.
 case "${ID}-${VERSION_ID}" in
-    "fedora-"*"")
+    "fedora-"*)
```
Comment (say-paul) [inline greenboot-test/greenboot-rs.sh]: not sure as why its failing here, the regex seems to be working for me in rhel

```
#!/bin/bash
ID="fedora"
VERSION_ID=40
case "${ID}-${VERSION_ID}" in
        "fedora-"*)
                echo "match"
                ;;
        *)

        echo "mismatch"
esac
```

Comment (say-paul) [reply greenboot-test/greenboot-rs.sh]: May be try after removing `"` , which also worked for me.

**Comment by yih-redhat** [comment]
Current status:
1. something wrong with say-paul/images:remove-old-greenboot-services, change to another image, and osbuild-composer can build successfully.
2. the greenboot rpm built from say-paul/greenboot branch greenboot-rs-continued caused osbuild-composer failed to build image, need to take look.

**Comment by yih-redhat** [comment]
/greenboot-rs-test-39 say-paul/images:remove-old-greenboot-services

**Comment by yih-redhat** [comment]
/greenboot-rs-test-39 say-paul/images:remove-old-greenboot-services

**Comment by yih-redhat** [comment]
/greenboot-rs-test-39 say-paul/images:remove-old-greenboot-services

**Comment by yih-redhat** [comment]
/greenboot-rs-test-39 say-paul/images:remove-old-greenboot-services

**Comment by yih-redhat** [comment]
/greenboot-rs-test-39 say-paul/images:remove-old-greenboot-services

**Pull request closed without merging**